controllers/activemqartemis_reconciler.go

```diff
@@ -2166,7 +2171,7 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) PodTemplateSpecForCR(customReso
 						},
 					},
 				},
-				InitialDelaySeconds:           1,
+				InitialDelaySeconds:           5,
```

Comment:
Is this change on purpose?

Reply:
yes but probably unnecessary, it related to my test env.  I will revert as it will limit availability on faster machines. good catch